.github/workflows/build.yml

```
@@ -124,11 +127,20 @@ jobs:
           PACKAGE_URL: honnef.co/go/tools/cmd/staticcheck
           PACKAGE_VERSION: ${{ steps.setup-env.outputs.staticcheck-version }}
         run: go install ${PACKAGE_URL}@${PACKAGE_VERSION}
-      - name: Install Terraform-docs
-        env:
-          PACKAGE_URL: github.com/terraform-docs/terraform-docs
-          PACKAGE_VERSION: ${{ steps.setup-env.outputs.terraform-docs-version }}
-        run: go install ${PACKAGE_URL}@${PACKAGE_VERSION}
+      # We are temporarily using @mcdonnnj's forked branch of terraform-docs
+      # until his PR: https://github.com/terraform-docs/terraform-docs/pull/745
+      # is approved. This temporary fix will allow for ATX Header Support when
+      # terraform-docs is ran during lint.
+      - name: Clone ATX Headers branch from Terraform-docs Fork
+        run: |
+          git clone --branch $BRANCH_NAME --single-branch \
+           --depth=$DEPTH $REPO_URL /tmp/terraform-docs
+      - name: Build and Install Terraform-docs Binary
+        run: |
+          cd /tmp/terraform-docs
+          GOBIN=$(go env GOPATH)/bin
+          go build -o $GOBIN/terraform-docs
+          echo "$GOBIN" >> $GITHUB_PATH
```

I believe the PATH is handled by [actions/setup-go](https://github.com/actions/setup-go/blob/0c52d547c9bc32b1aa3301fd7a9cb496313a4491/src/main.ts#L44). We can also take advantage of the `-C` switch to handle building from the cloned repository.

```suggestion
          go build \
          -C /tmp/terraform-docs \
          -o $(go env GOPATH)/bin/terraform-docs
```